src/main/groovy/org/mockito/release/notes/vcs/DefaultContributionSetSerializer.java

```diff
@@ -39,7 +35,7 @@ public DefaultContributionSet deserialize(JsonObject jsonObject) {
 
     private DefaultContributionSet addCommits(DefaultContributionSet defaultContributionSet, JsonArray commits) {
         for (int i = 0; i < commits.size(); i++) {
-            Commit commit = gitCommitSerializer.deserialize((JsonObject) commits.get(0));
+            Commit commit = gitCommitSerializer.deserialize((JsonObject) commits.get(i));
```

Comment:
@NagRock Yes. Is class `DefaultContributionSerializer` needed? It looks like only used in test code?